index.html

```diff
@@ -388,5 +389,34 @@ <h2>About me</h2>
             </div>
         </section>
     </main>
+    <footer class="footer">
+        <div class="footerHeader">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+
+        <div class="footerForm">
+            <form action="https://formspree.io/f/mgedowaj" method="POST">
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, you are to stick to the design as much as possible. Please fix the name and email input to match the original design and also edit the submit button by removing the black border and also edit the value of the submit button to match the original.😊😊
### Yours
![footer1](https://user-images.githubusercontent.com/63964646/149000177-113081be-7366-4888-878b-36638e457119.PNG)

### Figma Template
![footer2](https://user-images.githubusercontent.com/63964646/149000185-427a4bba-c144-4002-a81e-89c3575ef85d.PNG)

Reply:
Hello, @lily-coder  thank you for the review.
- I have made the required changes. 
- I however do not know why it is showing a black border on the button. I have cross checked my styling and there is not black border.

Reply:
Please be sure to address the required changes asked by the previous reviewer. Sadly, your design still has the same issues as before 😥

![image](https://user-images.githubusercontent.com/15807343/149015361-3bef7ce5-3140-48ed-b27e-3327f8b901df.png)

Reply:
Hello @mateo951  thank you for this review.
I have once again cross-checked my styling for the button and I have not found any set border of black. I have also noted that the black border only appears when viewed from a mobile device.
To remedy this, I have given the button class a border value of 0 and made the required changes and now there is no border appearing.